CodeStarterCamp_Week5/AuditionManager.swift

```diff
@@ -26,15 +26,10 @@ final class AuditionManager {
                 return
             }
             
-            guard talent.singing == .A else {
-                return
-            }
-            
-            guard talent.dancing == .A else {
-                return
-            }
-            
-            guard talent.acting == .A else {
+            guard talent.singing == .A,
+                  talent.dancing == .A,
+                  talent.acting == .A
+            else {
                 return
             }
```

Comment:
이것도 훌륭한 것 같습니다!
제가 의도했던건 아래와 같습니다 ㅎㅎ

```suggestion
            [talent.singing, talent.dancing, talent.acting].forEach {
                guard $0 == .A else { return }
            }
```